<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_add_book">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete_book">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update_book">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Book</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_author">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_price">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_number">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Please enter book information below first before updating</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_photo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Assignment 4/images/default.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh books</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
